cur/programming/2048-testing/intro_to_hw3.html

```diff
@@ -12,69 +12,51 @@
        </p>
 
-       <p>
-         Now that you've read over the basics of the assignment, let's refine our understanding. Begin this lab by opening up the starter file. You should see:
-           <p><img src="/bjc-r/img/2048-testing/main_sprite_code.png" alt="lists of nouns, verbs, etc." title="lists of nouns, verbs, etc." /></p>
-
-        <p>Click on the "Game Code" Sprite, and you should see a ton of code. <b>For this HW, you should NOT change any existing code in this sprite (otherwise you might break the game), </b> unless you are doing extra credit.
-
-        <p>Without changing any existing code, add a script to the scripting area of the Game Code sprite with the following code (the "update display for" block is under Looks). Run it, and you should see a 2 appear gloriously in the top right corner (position 1, 4). 
-        </p>
-           <p><img src="/bjc-r/img/2048-testing/simple_board_setup_script.png" alt="lists of nouns, verbs, etc." title="lists of nouns, verbs, etc." /></p>
-
-           <div class="alert quoteGreen alert-success">
-             <h4><strong>For You To Do:</strong></h4>
-             <p>
-               Modify the script above so that it displays the board pictured below. This is an opportunity for you to use the <img src="/bjc-r/img/2048-testing/set_item.png" /> block and learn how the grid works for the 2048 game.
-             </p>
-             <img src="/bjc-r/img/2048-testing/example_board.png" alt="this should be a board" style="width: 10em"/>
-
              <br /><br />
             <div class="assessment-data"  type="multiplechoice"  maxchoices="1"
                 responseIdentifier="2048" hasinlinefeedback="true"  shuffle="true">
                 <div class="prompt">
                     <p>
-                      In the above image, what is the position of the <strong>16</strong> tile?
+                      Exactly how many blocks are you supposed to implement for HW3?
                     </p>
                 </div>
 
                 <div class="choice" identifier="c1">
-                    <div class="text">(2, 3)</div>
+                    <div class="text">3</div>
                     <div class="feedback">
-                        Remember, that we specify positions in the format (row, column)
+                        Give the spec another look! It's important that you fully understand the assignment before starting this lab.
                     </div>
                 </div>
 
                 <div class="choice" identifier="c2">
-                    <div class="text">(2, 1)</div>
+                    <div class="text">1</div>
                     <div class="feedback">
-                        Consider what the starting value of the left box is. Is it 0 or 1?
+                      Give the spec another look! It's important that you fully understand the assignment before starting this lab.
```

Comment:
I think this feedback could be improved.

"It's important that you fully understand the assignment before starting this lab."
Generally telling people they need to "understand" or "know" more as part of an incorrect answer is taken the wrong wrong way, since it's pretty clear they don't understand something. I would suggest where they could look in the spec or give them something different to think about, like an aternate working, though an alternate wording probably isn't nedded for this question.